include/RAJA/util/PluginOptions.hpp

```diff
@@ -0,0 +1,31 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#ifndef RAJA_Plugin_Options_HPP
+#define RAJA_Plugin_Options_HPP
+
+namespace RAJA {
+namespace util {
+
+#include <string>
```

Comment:
Move this include prior to the namespace nesting. This can do very ugly things

Reply:
👍

Reply:
👍